go.mod

```diff
@@ -4,14 +4,17 @@ go 1.19
 
 require (
 	github.com/containerd/containerd v1.6.26
-	github.com/containerd/continuity v0.3.0
-	github.com/containerd/go-cni v1.1.6
+	github.com/containerd/continuity v0.4.3
+	github.com/containerd/go-cni v1.1.9
+	github.com/containerd/log v0.1.0
+	github.com/data-accelerator/zdfs v0.1.3
```

Comment:
The package lacks license
- https://github.com/data-accelerator/zdfs/issues/2

Reply:
Sorry...
I have added Apache license to the project.